PyInstaller/utils/hooks/qt.py

```diff
@@ -425,8 +427,8 @@ def get_qmake_path(version=''):
     "qt5sql":                   (".QtSql",                 "qtbase",           "sqldrivers"),
     "qt5svg":                   (".QtSvg",                 None,               ),
     "qt5test":                  (".QtTest",                "qtbase",           ),
-    "qt5webkit":                (None,                     None,               ),
-    "qt5webkitwidgets":         (None,                     None,               ),
+    "qt5webkit":                (".QtWebKit",              None,               ),
+    "qt5webkitwidgets":         (".QtWebKitWidgets",       None,               "qt5webkit"),
```

Comment:
@bjones I'm curious: is this change actually related to cygwin? Or related to some version of Qt?